### ISSUE TITLE
Have tests

### ISSUE THREAD

**Issue by justinabrahms** [alex/pyvcs#13]
srsly. Pull requests coming in w/o legitimate test cases sucks. Investigate some form of test suite.

**Comment by swilcox** [comment]
agreed. Just a little tricky since we're talking about hitting vcs's. Guessing that the tests will have to involve some shell scripts? Not sure...
